Sdk/Tasks/DmappCreation.cs

```diff
@@ -203,23 +203,26 @@ private bool PackageTests(PackageCreationData preparedData, AppPackageBuilder ap
             }
 
             // Handles all different special technologies that use non-sdk-project automationscripts for tests.
-            Logger.ReportWarning("Adding Harvested Automation XML Tests to .dmtest");
+            Logger.ReportStatus("Adding Harvested Automation XML Tests to .dmtest");
             AddHarvestedAutomationTests(pathToCustomTestHarvesting, appPackageBuilder);
 
             // Handles all other testing technologies
-            Logger.ReportWarning("Adding Harvested Tests to .dmtest");
+            Logger.ReportStatus("Adding Harvested Tests to .dmtest");
```

Comment:
Please verify if this logging is visible in the output. I remember vaguely that I only got it visible in MSBuild output when reporting it a warnings.